.changeset/nice-flies-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@directus/api": patch
+---
+
+Improve performance when importing files
```

Comment:
```suggestion
Moved file import to separate thread to prevent slow down of main thread
```

Reply:
Should we be more specific? Worker thread is the default from tinypool where child process is an alternative.
```suggestion
Prevent slow down of the main thread by moving file import to worker threads
```

Reply:
SGTM! Thanks @licitdev ❤️